cogs/database_bg.py

```diff
@@ -0,0 +1,43 @@
+from datetime import datetime
+from discord.ext import tasks,commands
+from coc import utils
+
+class Database_Background(commands.Cog):
+    def __init__(self, bot):
+        self.bot = bot
+        self.update.start()
+
+    @commands.command(name="add_user")
+    async def add_user(self, ctx, player_tag):
+        "Command is used to register a user to the database"
+        player_tag = utils.correct_tag(player_tag)
+        player = await self.bot.coc.get_player(player_tag)
+        commit = self.bot.dbconn.register_user((
+            player.tag,
+            player.name,
+            player.town_hall,
+        ))
+        if commit:
+            await ctx.send(f'Error: {commit}')
```

Comment:
Code block might make easier reading of the error.

```suggestion
            await ctx.send(f"Error: `{commit}`")
```